doc/reference/reference_lua/yaml.rst

```diff
@@ -57,6 +63,171 @@ Below is a list of all ``yaml`` functions and members.
     :return: the original contents formatted as a Lua table.
     :rtype: table
 
+.. _yaml-serialize:
+
+**__serialize parameter:**
+
+The YAML output structure can be specified with ``__serialize``:
+
+* 'seq', 'sequence', 'array' - table encoded as an array
+* 'map', 'mappping' - table encoded as a map
+* function - the meta-method called to unpack serializable representation
+  of table, cdata or userdata objects
+
+'seq' or 'map' also enable the flow (compact) mode for the YAML serializer
+(flow="[1,2,3]" vs block=" - 1\n - 2\n - 3\n").
+
+Serializing 'A' and 'B' with different ``__serialize`` values brings different
+results:
+
+.. code-block:: tarantoolsession
+
+    tarantool> yaml.encode(setmetatable({'A', 'B'}, { __serialize="seq"}))
+    ---
+    - '["A","B"]'
+    ...
+    tarantool> yaml.encode(setmetatable({'A', 'B'}, { __serialize="map"}))
+    ---
+    - '{"1":"A","2":"B"}'
+    ...
+    tarantool> yaml.encode({setmetatable({f1 = 'A', f2 = 'B'}, { __serialize="map"})})
+    ---
+    - '[{"f2":"B","f1":"A"}]'
+    ...
+    tarantool> yaml.encode({setmetatable({f1 = 'A', f2 = 'B'}, { __serialize="seq"})})
+    ---
+    - '[[]]'
+    ...
+
+.. _yaml-cfg:
+
+.. function:: cfg(table)
+
+    Set values affecting the behavior of encode and decode functions.
+
+    The values are all either integers or boolean ``true``/``false``.
+
+    .. container:: table
+
+        .. rst-class:: left-align-column-1
+        .. rst-class:: center-align-column-2
+        .. rst-class:: left-align-column-3
+
+        +---------------------------------+---------+-------------------------------------------+
+        | Option                          | Default | Use                                       |
+        +=================================+=========+===========================================+
+        | ``cfg.encode_invalid_numbers``  |  true   | A flag saying whether to enable encoding  |
+        |                                 |         | of NaN and Inf numbers                    |
+        +---------------------------------+---------+-------------------------------------------+
+        | ``cfg.encode_number_precision`` | 14      | Precision of floating point numbers       |
+        +---------------------------------+---------+-------------------------------------------+
+        | ``cfg.encode_load_metatables``  | true    | A flag saying whether to crop tables       |
+        |                                 |         |  with nesting level deeper than           | 
+        |                                 |         | ``max_depth``.                           |
```

Comment:
To avoid confusion, please specify the full name:

max_depth --> ``cfg.encode_max_depth`

But I don't see `cfg.encode_max_depth` in yaml.rst. Is it a bug?

Reply:
no, it's N/A for yaml

Reply:
here is another mistake. The description is wrong. I'll fix it